PhysicsTools/HeppyCore/python/framework/chain.py

```diff
@@ -6,6 +6,12 @@
 import pprint
 from ROOT import TChain, TFile, TTree, gSystem
 
+def is_lfn(fn):
+    return ":" in fn or fn.startswith("/store")
```

Comment:
uhm.... to my understanding lfns are only those that start with `/store`, and instead things like `root://eoscms//eos/cms/store/...` are pfns